x-pack/plugins/apm/public/plugin.ts

```diff
@@ -138,51 +136,6 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
   }
   public start(core: CoreStart, plugins: ApmPluginStartDeps) {
     toggleAppLinkInNav(core, this.initializerContext.config.get());
-
-    plugins.triggers_actions_ui.alertTypeRegistry.register({
-      id: AlertType.ErrorRate,
-      name: i18n.translate('xpack.apm.alertTypes.errorRate', {
-        defaultMessage: 'Error rate',
-      }),
-      iconClass: 'bell',
-      alertParamsExpression: lazy(
-        () => import('./components/shared/ErrorRateAlertTrigger')
-      ),
-      validate: () => ({
-        errors: [],
-      }),
-      requiresAppContext: true,
-    });
-
-    plugins.triggers_actions_ui.alertTypeRegistry.register({
-      id: AlertType.TransactionDuration,
-      name: i18n.translate('xpack.apm.alertTypes.transactionDuration', {
-        defaultMessage: 'Transaction duration',
-      }),
-      iconClass: 'bell',
-      alertParamsExpression: lazy(
-        () => import('./components/shared/TransactionDurationAlertTrigger')
-      ),
-      validate: () => ({
-        errors: [],
-      }),
-      requiresAppContext: true,
-    });
-
-    plugins.triggers_actions_ui.alertTypeRegistry.register({
-      id: AlertType.TransactionDurationAnomaly,
-      name: i18n.translate('xpack.apm.alertTypes.transactionDurationAnomaly', {
-        defaultMessage: 'Transaction duration anomaly',
-      }),
-      iconClass: 'bell',
-      alertParamsExpression: lazy(
-        () =>
-          import('./components/shared/TransactionDurationAnomalyAlertTrigger')
-      ),
-      validate: () => ({
-        errors: [],
-      }),
-      requiresAppContext: true,
-    });
+    registerApmAlerts(plugins.triggers_actions_ui.alertTypeRegistry);
```

Comment:
Moved all the registration into a separate file to mirror the server-side approach

Reply:
Looks clean 👍